drivers/ioexpander/gpio.c

```diff
@@ -596,6 +596,64 @@ int gpio_pin_register(FAR struct gpio_dev_s *dev, int minor)
   return register_driver(devname, &g_gpio_drvrops, 0666, dev);
 }
 
+/****************************************************************************
+ * Name: gpio_pin_register_byname
+ *
+ * Description:
+ *   Register GPIO pin device driver by it's name.
+ *
```

Comment:
Please include Input Parameters and returned value

Reply:
Done